packs/typeform/actions/get_results.py

```diff
@@ -0,0 +1,40 @@
+import urllib
+import httplib
+import requests
+
+from st2actions.runners.pythonrunner import Action
+
+__all__ = [
+    'TypeformAction'
+]
+
+
+class TypeformAction(Action):
+
+    def run(self, form_id, api_key, completed=True):
+        api_key = api_key if api_key else self.config['api_key']
+        completed = str(completed).lower()
+        url = "https://api.typeform.com/v0/form/%s?key=%s&completed=%s" % \
```

Comment:
https://docs.python.org/2/library/urlparse.html urljoin is better to use. it deals with '/' at the end and what not.